### PR TITLE
Prepare weaver tests for CI

### PR DESCRIPTION
**Pull request by fealebenpae**
This changes the `RealmWeaver.Tests` project so that it builds from the OS X Terminal. The goal is to build these from the command-line and run them with the NUnit Console Runner as part of CI.

### CODE REVIEW

File: Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs [hunk 1]
```diff
@@ -16,6 +16,9 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+extern alias realm;
```
Comment (kristiandupont) [inline Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs]: Is there some convention that I am unaware of stating that extern aliases are named with lowercase?

File: Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs [hunk 2]
```diff
@@ -135,42 +93,50 @@ private void WeavePropertyChanged(string targetAssemblyPath)
...
         public void FixtureSetup()
         {
-            var assemblyDir = GetAssemblyDir();
-            TryCleanupOldAssemblies(assemblyDir);
+            _sourceAssemblyPath = typeof(AssemblyToProcess.AllTypesObject).Assembly.Location;
+            _targetAssemblyPath = _sourceAssemblyPath.Replace(".dll", $".{_weaverOptions}.dll");
 
-            _sourceAssemblyPath = GetSourceAssemblyPath();
-            _targetAssemblyPath = CopyAssemblyToNextUniquePath(_sourceAssemblyPath);
+            if (File.Exists(_targetAssemblyPath))
```
Comment (kristiandupont) [inline Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs]: I believe this check is unnecessary?

**Comment by kristiandupont** [comment]
:+1:

File: Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs (hunk 1 above)
Comment (fealebenpae) [inline Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs]: Well, `global`, the default alias, is lowercase, but I don't know of a convention.

File: Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs (hunk 2 above)
Comment (fealebenpae) [inline Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs]: Yeah, I believe we can do without. It was left from an earlier iteration I had on this, but is now superfluous.

**Pull request merged**